localgov_microsites_group.info.yml

```diff
@@ -4,10 +4,11 @@ description: Group integration for Microsites.
 package: LocalGov Drupal
 core_version_requirement: ^9
 dependencies:
+  - field_group:field_group
   - group:group
+  - group:gnode
   - image:image
   - link:link
   - media_library:media_library
   - path:path
-  - field_group:field_group
   - localgov_core:localgov_media
```

Comment:
Do we need to add `localgovdrupal/localgovpage` to this module's composer.json?

Reply:
The config for this is in optional, and so we don't need to add an explicit dependency, the config will only get installed if localgov_page is enabled (which is done in the profile). This keeps the dependency chain simpler and shouldn't cause issues, but we could make the dependency explicit.

Reply:
Correlated with https://github.com/localgovdrupal/localgov_microsites_group/issues/13 that I just posted.

Is the assumption going to be that we always have localgovpage; and it will _always_ be enabled?
Or does it make sense to accept that people might not use it and have directory only groups or some such?